platform/qt/src/qmapboxgl.cpp

```diff
@@ -273,8 +362,28 @@ void QMapboxGLSettings::setAccessToken(const QString &token)
 */
 
 /*!
+    \enum QMapboxGL::NorthOrientation
+
+    This enum sets the orientation of the north heading. It will directly affect bearing when
```

Comment:
@lyzidiamond `This enum ...` is actually a standard phrasing on the Qt docs. See example: http://doc.qt.io/qt-5/qserialport.html